### PR TITLE
Switched to flow isolation by default instead of dropping from queue 0

### PR DESCRIPTION
**Pull request by cliffburdick**

### CODE REVIEW

File: applications/adv_networking_bench/adv_networking_bench_default_rx.yaml [hunk 1]
```diff
@@ -48,28 +48,15 @@ advanced_network:
...
-          - name: "Default"
-            id: 0
-            cpu_core: 7
-            batch_size: 10240
-            output_port: "bench_rx_out"
-            memory_regions:
-              - "Default_RX_CPU"
           - name: "Data"
-            id: 1
+            id: 0
```
Comment (agirault) [inline applications/adv_networking_bench/adv_networking_bench_default_rx.yaml:59]: then need to update queue id in the ADC Samples flow also

File: applications/adv_networking_bench/adv_networking_bench_default_rx.yaml [hunk 2]
```diff
@@ -48,28 +48,15 @@ advanced_network:
         - local
       num_bufs: 30720
       buf_size: 1064
-    - name: "Default_RX_CPU"
```
Comment (agirault) [inline applications/adv_networking_bench/adv_networking_bench_default_rx.yaml:51]: needed by `Data` queue for HDS

Comment (cliffburdick) [reply applications/adv_networking_bench/adv_networking_bench_default_rx.yaml:51]: Actually, no, this is not needed. Data_RX_CPU is used for HDS

File: applications/adv_networking_bench/adv_networking_bench_default_tx_rx.yaml [hunk 3]
```diff
@@ -42,13 +42,6 @@ advanced_network:
         - local
       num_bufs: 51200
       buf_size: 9000
-    - name: "Default_RX_GPU"
```
[the remark this answers is not in the document]
Comment (cliffburdick) [reply applications/adv_networking_bench/adv_networking_bench_default_tx_rx.yaml:45]: It has HDS with Data_RX_CPU/Data_RX_GPU

File: applications/adv_networking_bench/adv_networking_bench_default_rx.yaml (hunk 2 above)
Comment (agirault) [reply applications/adv_networking_bench/adv_networking_bench_default_rx.yaml:51]: my bad

**Review by agirault — APPROVED** [review]

**Pull request merged**